Makefile.Crossdock.mk

```diff
@@ -32,20 +32,5 @@ build-crossdock-fresh: build-crossdock-linux
 	make crossdock-fresh
 
 .PHONY: crossdock-docker-images-jaeger-backend
-crossdock-docker-images-jaeger-backend: PLATFORMS=linux/amd64
-crossdock-docker-images-jaeger-backend: create-baseimg create-debugimg
-	for component in "jaeger-agent" "jaeger-collector" "jaeger-query" "jaeger-ingester" "all-in-one" ; do \
-		regex="jaeger-(.*)"; \
-		component_suffix=$$component; \
-		if [[ $$component =~ $$regex ]]; then \
-			component_suffix="$${BASH_REMATCH[1]}"; \
-		fi; \
-		docker buildx build --target $(TARGET) \
-			--tag $(DOCKER_NAMESPACE)/$$component$(SUFFIX):${DOCKER_TAG} \
-			--build-arg base_image=$(BASE_IMAGE) \
-			--build-arg debug_image=$(DEBUG_IMAGE) \
-			--build-arg TARGETARCH=$(GOARCH) \
-			--load \
-			cmd/$$component_suffix; \
-		echo "Finished building $$component ==============" ; \
-	done;
+crossdock-docker-images-jaeger-backend: docker-images-jaeger-backend
+	make docker-images-jaeger-backend
```

Comment:
since you already made the target dependent on `docker-images-jaeger-backend` there is no need to call make explicitly again

Reply:
Ahh makes sense. Sorry using Makefile for the first time.